### PR TITLE
Additional features and options

### PR DESCRIPTION
**Pull request by alexby**
Added:
-docker container
-options to export not only `image` and `thumbnail`, but also custom fields

Fixed
-readme
-removed limitation of 1000 files to export

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,7 @@ This command will read from a SAS Parse server, upload pictures to a given S3 bu
 
 
 ```bash
-git clone git@github.com:Meetic/ParseServerPicturesMigration.git
-cd ParseServerMigrationTool
-composer install
+git clone git@github.com:alexby/parse-files-migration-tool.git migration-tool
```
Comment (alexby) [inline README.md]: Yeah, I left my url in my fork. Feel free to change it back in your original repo.

File: application.php [hunk 2]
```diff
@@ -2,6 +2,8 @@
 <?php
 // application.php
 
+ini_set('memory_limit', '-1');
```
Comment (alexby) [inline application.php]: Yes, I had troubles if big number of files should be transfered.

File: src/ParseServerMigration/Console/Command/DeleteCommand.php [hunk 3]
```diff
@@ -66,11 +67,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             try {
                 $deleteResult = $this->pictureRepository->deletePicture($picture);
 
-                $message = 'Export success for: ['.$picture->get('image')->getName().'] Uploaded to : ['.$deleteResult['ObjectURL'].']';
+                $message = 'Export success for: ['.$picture->get(Config::PARSE_FILES_FIELD_NAME)->getName().'] Uploaded to : ['.$deleteResult['ObjectURL'].']';
```
Comment (alexby) [inline src/ParseServerMigration/Console/Command/DeleteCommand.php]: Thanks :)

File: README.md (hunk 1 above)
Comment (MaxenceDupressoir) [inline README.md]: Yep I'll do it :)

**Pull request merged**